lib/Common/Memory/LargeHeapBlock.h

```diff
@@ -258,6 +258,19 @@ class LargeHeapBlock sealed : public HeapBlock
 
     bool isInPendingDisposeList;
 
+#ifdef RECYCLER_PAGE_HEAP
+    PageHeapMode pageHeapMode;
+    char* guardPageAddress;
+    StackBackTrace* pageHeapAllocStack;
+    StackBackTrace* pageHeapFreeStack;
+
+public:
+    __inline bool InPageHeapMode() const { return pageHeapMode != PageHeapMode::PageHeapModeOff; }
+    void CapturePageHeapAllocStack();
+    void CapturePageHeapFreeStack();
+    const StackBackTrace* failedAllocStackTracePtr = (StackBackTrace*)1;
```

Comment:
Nit: consider making this static const and prefix with s_. In any case, change this to StackTraceAllocFailed since it's not a real ptr and constants are PascalCased.